src/main/java/ladder/ui/ResultView.java

```diff
@@ -0,0 +1,43 @@
+package ladder.ui;
+
+import ladder.domain.ladder.Ladder;
+import ladder.domain.line.Line;
+import ladder.domain.player.Player;
+import ladder.utils.StringUtil;
+
+import java.util.stream.Collectors;
+
+public class ResultView {
+
+	private static final String RESULT_MESSAGE = "\n실행결과\n";
+	private static final String VERTICAL_DELIMITER = "|";
+	private static final String VERTICAL_DELIMITER_PREFIX = "  |";
+	private static final String LINE_DASH = "-----";
+	private static final String LINE_EMPTY = "     ";
+
+	public static void printLadder(Ladder ladder) {
+		printPlayers(ladder);
+
+		ladder.getLines().stream()
+		      .map(ResultView::lineToDash)
+		      .collect(Collectors.toList())
+		      .forEach(System.out::println);
```

Comment:
```suggestion
		ladder.getLines().stream()
		      .map(ResultView::lineToDash)
		      .forEach(System.out::println);
```
단순히 출력한다면 list로 만들지 않아도 될 것 같습니다 :)